rdmd.d

```diff
@@ -708,10 +708,7 @@ private string[string] getDependencies(string rootModule, string workDir,
     }
 
     if (!needInspection)
-    {
-        string[string] noDeps;
-        return noDeps;
-    }
+        return null;
```

Comment:
To clarify, the added and removed code is identical in functionality. An associative array is initialized to `null` by default.